src/include/cell_inline.h

```diff
@@ -943,9 +947,9 @@ __wt_cell_unpack_safe(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CE
     case WT_CELL_ADDR_LEAF_NO:
         /* Unpack the row and/or byte counts if the chunk of data includes it. */
         if (ps != NULL && __wt_process.page_stats_2022) {
-            if (F_ISSET(dsk, WT_PAGE_STAT_BYTE_COUNT))
+            if (cell->__chunk[0] & WT_CELL_STAT_BYTE_COUNT)
```

Comment:
If we pack the stat before page del information, this should also be unpacked before page_del